pkg/sse/stream_events.go

```diff
@@ -60,9 +63,25 @@ func (sr *StreamRequest) Stream() StreamResponse {
 	}
 
 	data := bufio.NewScanner(resp.Body)
+	s := spinner.New(spinner.CharSets[14], 100*time.Millisecond)
 	for data.Scan() {
-		line := data.Bytes()
-		logger.Output(string(line))
+		line := string(data.Bytes())
+		if line == "" {
+			continue
+		}
+		if strings.Contains(line, ui.SPINNER) {
+			parts := strings.Split(line, ui.SPINNER)
+			s.Prefix = parts[0]
+			s.Suffix = parts[1]
+			err := s.Color("fgHiBlue", "bold")
```

Comment:
Use this color from constants instead of hardcoding here

Reply:
done